src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Transaction;
+import java.util.prefs.BackingStoreException;
+
+public interface OperationHandler {
+    void apply(Transaction transaction) throws BackingStoreException;
```

Comment:
```suggestion
    void apply(Transaction transaction);
```
throw RuntimeException inside implementation where needed